app/models/manageiq/providers/azure/cloud_manager.rb

```diff
@@ -45,6 +45,10 @@ def insights?
     end
   end
 
+  def case_insensitive_labels?
+    true
```

Comment:
This should be `case_sensitive_labels? false`

Reply:
yes, it was leftover and I removed it now.

we have such method on azure's  persister

thanks